resources/views/service/centres/create.blade.php

```diff
@@ -31,12 +30,13 @@
                     </select>
                 </div>
                 <div class="select">
-                    <label for="form">Form</label>
-                    <select name="form" id="form" class="{{ $errors->has('form') ? 'error' : '' }}" required>
-                        <option value="">Choose one</option>
+                    <label for="print_pref">Printed Form</label>
+                    <select name="print_pref" id="print_pref" class="{{ $errors->has('print_pref') ? 'error' : '' }}" required>
+                        <option value="" disabled selected>Choose one</option>
```

Comment:
Text outside the quotes

Reply:
yep, they're the null attributes `disabled` and `selected`, so users must  choose a valid one and and it's not picked a default.

Reply:
:man_facepalming: Of course!